wlauto/instrumentation/poller/poller.yaml

```diff
@@ -0,0 +1,8 @@
+config:
```

Comment:
Rather than having this in a separate file that no one will look at -- user's aren't expected to look thorough the code, this should be part of the description. Also, I would remove everything not germane to the example (e.g. your current example can be taken to imply that this only works with android devices).